src/host/PtySignalInputThread.cpp

```diff
@@ -191,12 +209,9 @@ void PtySignalInputThread::CreatePseudoWindow()
             break;
         }
         default:
-        {
-            THROW_HR(E_UNEXPECTED);
-        }
+            return S_OK;

```

Comment:
We can't throw an exception here - no one is catching it and it would cross an ABI boundary otherwise. This is also why this function is now `noexcept` - it should've always been.

Reply:
You could wrap the whole function in a `try` block and add a `CATCH_LOG_RETURN_HR(S_OK);` at the end. Then just keep the `default` case as a `THROW_HR(E_UNEXPECTED)`.

Looks a bit ugly, but it's true, the `default:` case is definitely unexpected.